### DISCUSSION TITLE
Kubernetes- ReadOnly Root fs

### DISCUSSION THREAD

**Discussion by ruthcamarillo** [community/community#137504] [Actions]
### Select Topic Area

General

### Body

Hello, I am updating the readOnly root filesystem to be enabled, and I was able to deploy on the cdi-operator but my other cdi components were not updated. Does anyone have any solution to update my other components? I have looked around and read about customizedComponents patchs but wanted to check if anyone else had better solutions?

**Comment by Mrsanyi123** [comment]
Hello, @ruthcamarillo 
The answer to your question might be one of those!

1. Double-check the Customized Components Patch: Make sure it covers all CDI components to propagate the changes.

2. Update the Manifests: Manually update or use Helm to apply the read-only setting across all CDI components.
 
3. Reapply the CDI CRD: This might force a re-deployment with the new configuration.
 
4. Check Your Operator Version: Ensure you're using a version that supports your changes fully.
 
5. Inspect Logs: Look for any errors in the CDI Operator and components that might explain the issue.
 
6. Manual Rollout: If needed, manually restart the components to apply the updates.
 **I hope this helps @ruthcamarillo**
